### PR TITLE
Add netflow, ping and influx sections to docker-compose.yml

### PR DESCRIPTION
**Pull request by enumag**
Replace #53.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,10 @@ services:
     image: elasticsearch:2
     restart: always
 
+  influxdb:
+    image: influxdb:0.13-alpine
+    restart: always
```
Comment (enumag) [inline docker-compose.yml]: Volume is missing. Also need to add it to update.sh.

Comment (andreas4all) [inline docker-compose.yml]: volume in yml must be defined - for new users.

File: docker-compose.yml [hunk 2]
```diff
@@ -55,3 +59,14 @@ services:
     links:
       - postgresql
     command: "crm_draft_approve"
+
+  crm_netflow:
+    image: ubnt/ucrm-billing:latest
+    restart: always
+    env_file: docker-compose.env
```
Comment (andreas4all) [inline docker-compose.yml]: add volume section.

File: update.sh [hunk 3]
```diff
@@ -53,6 +53,20 @@ if [ $? = 1 ]; then
 	sed -i -e "s/      - postgresql/&\n      - elastic/g" docker-compose.yml
 fi
 
+cat -vt docker-compose.yml | egrep "  influxdb:" > /dev/null
+
+if [ $? = 1 ]; then
+	echo "Your docker-compose doesn't contain Influxdb section. Trying to add."
+	echo -e "\n\n  influxdb:\n    image: influxdb:0.13-alpine\n    restart: always" >> docker-compose.yml
```
Comment (andreas4all) [inline update.sh]: volume

File: update.sh [hunk 4]
```diff
@@ -53,6 +53,20 @@ if [ $? = 1 ]; then
...
+fi
+
+cat -vt docker-compose.yml | egrep "  crm_netflow:" > /dev/null
+
+if [ $? = 1 ]; then
+	echo "Your docker-compose doesn't contain Netflow section. Trying to add."
+	echo -e "\n\n  crm_netflow:\n    image: ubnt/ucrm-billing:latest\n    restart: always\n    env_file: docker-compose.env\n    links:\n      - postgresql\n      - influxdb\n    ports:\n      - 2055:2055\n    command: \"crm_netflow\"" >> docker-compose.yml
```
Comment (andreas4all) [inline update.sh]: volume

File: docker-compose.yml (hunk 2 above)
Comment (enumag) [inline docker-compose.yml]: Added by @keksa

File: docker-compose.yml (hunk 1 above)
Comment (enumag) [inline docker-compose.yml]: Added by @keksa

File: update.sh [hunk 5]
```diff
@@ -53,6 +19,27 @@ if [ $? = 1 ]; then
...
+if [ $? = 1 ]; then
+	echo "Your docker-compose doesn't contain Influxdb section. Trying to add."
+	echo -e "\n  influxdb:\n    image: influxdb:0.13-alpine\n    restart: always\n    volumes:\n      - ./data/influxdb:/var/lib/influxdb" >> docker-compose.yml
+fi
+
+cat -vt docker-compose.yml | egrep "  crm_netflow:" > /dev/null
+
+if [ $? = 1 ]; then
+	echo "Your docker-compose doesn't contain Netflow section. Trying to add."
+	echo -e "\n  crm_netflow:\n    image: ubnt/ucrm-billing:latest\n    restart: always\n    env_file: docker-compose.env\n    volumes:\n      - ./data/ucrm:/data\n    links:\n      - postgresql\n      - influxdb\n    ports:\n      - 2055:2055\n    command: \"crm_netflow\"" >> docker-compose.yml
```
Comment (UBNT-Petr) [inline update.sh]: 2055:2055/udp

**Comment by keksa** [comment]
closing in favour of #66

**Pull request closed without merging**